docs/source/api/iohub/device/eyetracker_interface/PupilLabs_Core_Implementation_Notes.rst

```diff
@@ -0,0 +1,464 @@
+#################
+Pupil Labs - Core
+#################
+
+.. contents:: Table of Contents
+
+**********************************
+High Level Pupil Core Introduction
+**********************************
+
+`Pupil Core <https://pupil-labs.com/products/core/>`__ is a wearable eye tracker. The
+system consists of two inward-facing eye cameras and a forward-facing world camera
+mounted on a wearable eyeglasses-like frame.
+
+Pupil Core provides gaze in its world camera's field of view, regardless of the wearer's
+head position. As such, gaze can be analysed with the wearer looking and moving freely
+in their environment.
+
+Pupil Core differs from remote eye trackers often used with PsychoPy. Remote eye
+trackers employ cameras mounted on or near a computer monitor. They provide gaze in
+screen-based coordinates, and this facilitates closed-loop analyses of gaze based on the
+known position of stimuli on-screen and eye gaze direction.
+
+.. image:: /images/pupil-core-render.jpg
+    :width: 700px
+    :align: center
+    :alt: Pupil Core binocular headset with high-speed scene camera
+
+The main goal of this Pupil Core-PsychoPy integration is to provide similar functionality, by transforming Pupil Core gaze from world camera to screen-based coordinates *via*
```

Comment:
I feel like this should be moved up to L:23 so that there is a clear problem statement/reference to remote systems, then state what we are trying to do with the Pupil Core PsychoPy integration.

In order to use Pupil Core for screen-based work in PsychoPy, you will need to be able to robustly locate a screen within the world camera field of view. We use a marker based approach using April Tags, which enables you to detect and track the screen and transform Pupil Core's gaze data from world camera based coordinates to screen-based coordinates.

Reply:
@N-M-T do you have any suggestions/response to this?